docs/howto/catalog-queries.md

```diff
@@ -33,10 +33,12 @@ curl -k https://localhost:8443/catalogs/operatorhubio/api/v1/all | <query>
     jq -s '.[] | select( .schema == "olm.package")'
     ```
 
-* Packages that support `AllNamespaces` install mode and do not use webhooks:
-    ``` terminal
-    jq -c 'select(.schema == "olm.bundle") | {"package":.package, "version":.properties[] | select(.type == "olm.bundle.object").value.data |  @base64d | fromjson | select(.kind == "ClusterServiceVersion" and (.spec.installModes[] | select(.type == "AllNamespaces" and .supported == true) != null) and .spec.webhookdefinitions == null).spec.version}'
-    ```
+Packages that support `AllNamespaces` install mode and do not use webhooks
+
+: 
+``` terminal
+jq -cs '[.[] | select(.schema == "olm.bundle" and (.properties[] | select(.type == "olm.csv.metadata").value.installModes[] | select(.type == "AllNamespaces" and .supported == true)) and .spec.webhookdefinitions == null) | .package] | unique[]'
+```
```

Comment:
There was a chance in formatting in this doc which it seems got lost during the rebase. It should be something like this:

```suggestion
* Packages that support `AllNamespaces` install mode and do not use webhooks:
    ``` terminal
    jq -cs '[.[] | select(.schema == "olm.bundle" and (.properties[] | select(.type == "olm.csv.metadata").value.installModes[] | select(.type == "AllNamespaces" and .supported == true)) and .spec.webhookdefinitions == null) | .package] | unique[]'
    ```
```